dnplab/dnpIO/specman.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import os
+from dnplab import dnpdata
+
+
+def import_specman(path):
+    """
+    Import specman data and return dnpdata object
+
+    Args:
+        path (str) : Path to either .d01 or .exp file
+
+    Returns:
+        specman_data (object) : dnpdata object containing specman data
+    """
+    if path[-1] == os.sep:
```

Comment:
This is potentially error-prone. In windows `os.sep == \` but `path[-1]` can still be `/` 
```suggestion
    path = os.path.normpath(path)
```
Need tests

Reply:
I added normpath to the wrapper function so the path is normalized before it gets to any of the IO functions